RELEASE_NOTES.rst

```diff
@@ -1,3 +1,95 @@
+Coala 0.10 - Polar Bear
```

Comment:
oh... I missed it too :(
@Mixih you are one step closer to getting fired from being a maintainer :P